features/location/impl/src/main/kotlin/io/element/android/features/location/impl/show/ShowLocationView.kt

```diff
@@ -138,12 +137,12 @@ fun ShowLocationView(
             state.description?.let {
                 Text(
                     text = it,
-                    textAlign = TextAlign.Center,
+                    textAlign = TextAlign.Justify,
```

Comment:
I'd just use `TextAlign.Start` here, justify looks a bit weird, as @csmith mentioned.

Reply:
TextAlign.Start doesn't change things much:
<img width="404" alt="Screenshot 2023-07-20 at 09 33 46" src="https://github.com/vector-im/element-x-android/assets/1273124/385a9823-542f-47f9-bced-0a3bab06c57b">
<img width="406" alt="Screenshot 2023-07-20 at 09 33 41" src="https://github.com/vector-im/element-x-android/assets/1273124/e57224af-32a2-45d0-a434-f791e8e3ca18">
<img width="402" alt="Screenshot 2023-07-20 at 09 33 38" src="https://github.com/vector-im/element-x-android/assets/1273124/8c61133f-4f12-43f7-9568-d351a4aef000">

Reply:
Oh sorry, I misunderstood it, you *always* want it centered, even if the text has several lines and overflows. It seems a bit weird to me, but in that case we should go back to center aligning the text and trying to figure out why we have that extra indentation instead.